client/my-sites/site-settings/media-settings/style.scss

```diff
@@ -0,0 +1,8 @@
+.media-settings__module-settings.is-indented {
+	margin-left: 32px;
```

Comment:
This can benefit from #11240, which is an effort to make all spacing between toggles and labels consistent.